forem/forem
# Ensure spam doesn't hit logged-out /latest

## What type of PR is this? (check all applicable)


## Description

We want to suppress low-quality content from search engines. The `/latest` list of articles should use `home_feed_minimum_score` instead of a hard-coded low score.

We also want to include a message about the suppressed content, prompting visitors to sign-in:

<img width="1242" alt="Screen Shot 2023-02-13 at 11 40 26" src="https://user-images.githubusercontent.com/2077/218440487-7248f044-00a0-46b8-988d-df4b3344e93e.png">



## Related Tickets & Documents



- Closes #18979

## QA Instructions, Screenshots, Recordings

Note that the `home_feed_minimum_score` scope also includes `featured` articles (this _might_ be an unanticipated change, there's some discussion of this scope in [this PR](https://github.com/forem/forem/pull/15850)).

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media3.giphy.com/media/I9BrUZDq5hksw/giphy.gif?cid=ecf05e47prk9wnj9vmmp90w3jfockj79eyimgvk2zefl0uc4&rid=giphy.gif&ct=g)
